File: src/main/java/org/funivan/intellij/FastCoddy/Helper/FileHelper.java
package org.funivan.intellij.FastCoddy.Helper;


import org.jetbrains.annotations.Nullable;

import java.io.File;
import java.io.FileReader;
import java.io.IOException;

/**
 * @author Ivan Scherbak <dev@funivan>
 */
public class FileHelper {

    /**
     * Helper function for reading file
     */
    @Nullable
    public static String getFileContent(String filename) {
        String content = null;
        File file = new File(filename);

        if (!file.exists()) {
            return null;
        }

        try {
            FileReader reader = new FileReader(file);
            char[] chars = new char[(int) file.length()];
            reader.read(chars);
            content = new String(chars);
            reader.close();
        } catch (IOException e) {
            e.printStackTrace();
        }
        return content;
    }
}

File: src/main/kotlin/org/funivan/intellij/FastCoddy/Helper/FileHelper.kt
package org.funivan.intellij.FastCoddy.Helper

import java.io.File
import java.io.FileReader
import java.io.IOException

/**
 * @author Ivan Shcherbak <alotofall@gmail.com>
 */
object FileHelper {
    /**
     * Helper function for reading file
     */
    fun getFileContent(filename: String): String {
        var content: String = ""
        val file = File(filename)
        if (!file.exists()) {
            return ""
        }
        try {
            val reader = FileReader(file)
            val chars = CharArray(file.length().toInt())
            reader.read(chars)
            content = String(chars)
            reader.close()
        } catch (e: IOException) {
            e.printStackTrace()
        }
        return content
    }
}
